pkg/apis/servicecatalog/types.go

```diff
@@ -360,3 +360,8 @@ const (
 	// BindingConditionReady represents a BindingCondition is in ready state.
 	BindingConditionReady BindingConditionType = "Ready"
 )
+
+// These are internal finalizer values to service catalog, must be qualified name.
+const (
+	FinalizerServiceCatalog string = "kubernetes-incubator/service-catalog"
```

Comment:
no, we need to define in internal name and the v1alpha1 form of the name.  in future iterations, we may change the name during conversion across versions.